components-api/src/main/scala/pl/touk/nussknacker/engine/api/component/SingleComponentConfig.scala

```diff
@@ -14,6 +14,14 @@ import pl.touk.nussknacker.engine.api.definition.{ParameterEditor, ParameterVali
   def paramConfig(name: String): ParameterConfig = params.flatMap(_.get(name)).getOrElse(ParameterConfig.empty)
 }
 
+@JsonCodec case class SingleComponentUIConfig(params: Option[Map[String, ParameterConfig]], icon: Option[String], docsUrl: Option[String], disabled: Boolean = false) {
```

Comment:
I thought that `SingleComponentConfig` is "UI". `componentGroup` is for the presentation purpose. Also, we shouldn't expose `componentGroup` in the `AdditionalComponentsUIConfigProvider` interface? I think that we need all but `componentId`.